instrumentation/otel-webserver-module/build.gradle

```diff
@@ -360,83 +366,98 @@ task assembleApacheModule(type: Tar) {
 }
 
 // NGINX
-task extractNginx(type: Copy) {
-    from "build-dependencies"
-    into buildDir
+task downloadNginx(type: Download) {
+    nginxSupportedVersions.each {
+        nginxVersion ->
+            src "http://nginx.org/download/nginx-${nginxVersion}.tar.gz"
```

Comment:
Atm we were building nginx from the source code we provided with Dockerfile right?